awx/main/utils/ansible.py

```diff
@@ -87,19 +87,25 @@ def could_be_inventory(project_path, dir_path, filename):
 
     # Filter files that do not use a character set consistent with
     # Ansible inventory mainly
+    matched = False
     try:
         # only read through first 10 lines for performance
-        with codecs.open(
+        counter = 0
+        with open(
             inventory_path,
-            'r',
-            encoding='utf-8',
-            errors='ignore'
+            encoding='utf-8'
         ) as inv_file:
-            for line in islice(inv_file, 10):
-                if not valid_inventory_re.match(line):
-                    return None
+            for line in inv_file:
```

Comment:
Instead of tracking a counter, you can do:

```python
for i, line in enumerate(inv_file):
    if i > 10:
        break
```

Reply:
*or* you could probably do this instead:

```python
for line in inv_file[:10]:
    ...
```

Reply:
Thanks Ryan. I used your first suggestion and removed the counter.